### DISCUSSION TITLE
PlugNmeet not works. Error: could not establish pc connection

### DISCUSSION THREAD

**Discussion by ChristianKJ** [mynaparrot/plugNmeet-server#361] [Q&A]
Good morning community. I have a problem with plugNmeet. I install everything correctly from the following guide https://www.plugnmeet.org/docs/developer-guide/setup-development.
The program starts correctly, I can log in, create the room, select the microphone and then it gives me this error.
<img width="271" alt="image" src="https://github.com/mynaparrot/plugNmeet-server/assets/95303392/4452fb29-912d-4f38-b40c-d504baf36682">

<img width="960" alt="image" src="https://github.com/mynaparrot/plugNmeet-server/assets/95303392/b3c271e6-090b-420a-9ba4-c4f064d66770">

I'm using WSL 2 ubuntu 22.04 on windows 11, could this be the problem?

**Comment by jibon57** [comment]
Give a try this: https://www.plugnmeet.org/docs/developer-guide/setup-development#troubleshooting

**Reply by ChristianKJ** [reply]
which IP should I put? The local IP of the wsl?

**Reply by jibon57** [reply]
I don' have experience to use windows or wsl. I even don't know from where you're trying to access. If your hostname isn't localhost then you must need to setup SSL. If you're accessing inside ubuntu then you can get IP by `ifconfig`  or `ip route get 8.8.8.8 | awk -F "src " 'NR==1{split($2,a," ");print a[1]}'`

**Reply by ChristianKJ** [reply]
i did but it still gives the exact same error

**Reply by Resul0316** [reply]
So is it resolved? Because I have the same problem. If resolved, how? I followed the same steps as it is indicated in documentation (--config "/app/livekit.yaml" --dev --node-ip YOUR_IP_HERE) but doesn't work.
@ChristianKJ @jibon57 @wbobteng

**Reply by Resul0316** [reply]
Moreover, I am adding here log:


`server-plugnmeet-api-1    | 08:21:53 | 204 |      99.239µs | 172.18.0.1 | OPTIONS | /auth/room/isRoomActive | -
server-plugnmeet-api-1    | 08:21:53 | 200 |   10.557844ms | 172.18.0.1 | POST | /auth/room/isRoomActive | -
server-plugnmeet-api-1    | 08:21:53 | 204 |      41.651µs | 172.18.0.1 | OPTIONS | /auth/room/create | -
server-livekit-1          | 2024-02-27T08:21:53.752Z    INFO    livekit service/roomallocator.go:138    selected node for room  {"room": "room01", "roomID": "RM_9okKDA5MadUb", "selectedNodeID": "ND_LarFVep4LH9V"}
server-livekit-1          | 2024-02-27T08:21:53.753Z    DEBUG   livekit routing/signal.go:100   starting signal connection      {"room": "room01", "reqNodeID": "ND_LarFVep4LH9V", "participant": "", "connID": "CO_PDPLKL3tCcQx"}
server-livekit-1          | 2024-02-27T08:21:53.754Z    DEBUG   livekit service/signal.go:168   signal stream closed    {"room": "room01", "participant": "", "connID": "CO_PDPLKL3tCcQx", "error": null}
server-livekit-1          | 2024-02-27T08:21:53.754Z    DEBUG   livekit routing/signal.go:136   signal stream closed    {"room": "room01", "reqNodeID": "ND_LarFVep4LH9V", "participant": "", "connID": "CO_PDPLKL3tCcQx", "error": "stream closed"}
server-livekit-1          | 2024-02-27T08:21:53.754Z    INFO    livekit.api     service/twirp.go:124    API RoomService.CreateRoom      {"service": "RoomService", "method": "CreateRoom", "room": "room01", "request": "name:\"room01\"  empty_timeout:7200  metadata:\"{\\\"room_title\\\":\\\"Demo room\\\",\\\"welcome_message\\\":\\\"Welcome to plugNmeet!<br /> To share microphone click mic icon from bottom left side.\\\",\\\"is_recording\\\":false,\\\"is_active_rtmp\\\":false,\\\"parent_room_id\\\":\\\"\\\",\\\"is_breakout_room\\\":false,\\\"started_at\\\":\\\"1709022113\\\",\\\"room_features\\\":{\\\"allow_webcams\\\":true,\\\"mute_on_start\\\":false,\\\"allow_screen_share\\\":true,\\\"allow_rtmp\\\":true,\\\"allow_view_other_webcams\\\":true,\\\"allow_view_other_users_list\\\":true,\\\"admin_only_webcams\\\":false,\\\"allow_polls\\\":true,\\\"room_duration\\\":\\\"0\\\",\\\"enable_analytics\\\":true,\\\"allow_virtual_bg\\\":true,\\\"allow_raise_hand\\\":true,\\\"recording_features\\\":{\\\"is_allow\\\":true,\\\"is_allow_cloud\\\":true,\\\"enable_auto_cloud_recording\\\":false,\\\"is_allow_local\\\":true,\\\"only_record_admin_webcams\\\":false},\\\"chat_features\\\":{\\\"allow_chat\\\":true,\\\"allow_file_upload\\\":true,\\\"allowed_file_types\\\":[\\\"jpg\\\",\\\"png\\\",\\\"zip\\\"],\\\"max_file_size\\\":\\\"10\\\"},\\\"shared_note_pad_features\\\":{\\\"allowed_shared_note_pad\\\":true,\\\"is_active\\\":false,\\\"visible\\\":false,\\\"node_id\\\":\\\"\\\",\\\"host\\\":\\\"\\\",\\\"note_pad_id\\\":\\\"\\\",\\\"read_only_pad_id\\\":\\\"\\\"},\\\"whiteboard_features\\\":{\\\"allowed_whiteboard\\\":true,\\\"visible\\\":false,\\\"whiteboard_file_id\\\":\\\"\\\",\\\"file_name\\\":\\\"\\\",\\\"file_path\\\":\\\"\\\",\\\"total_pages\\\":0},\\\"external_media_player_features\\\":{\\\"allowed_external_media_player\\\":true,\\\"is_active\\\":false},\\\"waiting_room_features\\\":{\\\"is_active\\\":true,\\\"waiting_room_msg\\\":\\\"\\\"},\\\"breakout_room_features\\\":{\\\"is_allow\\\":true,\\\"is_active\\\":false,\\\"allowed_number_rooms\\\":6},\\\"display_external_link_features\\\":{\\\"is_allow\\\":true,\\\"is_active\\\":false},\\\"ingress_features\\\":{\\\"is_allow\\\":true,\\\"input_type\\\":\\\"RTMP_INPUT\\\",\\\"url\\\":\\\"\\\",\\\"stream_key\\\":\\\"\\\"},\\\"speech_to_text_translation_features\\\":{\\\"is_allow\\\":false,\\\"is_allow_translation\\\":true,\\\"is_enabled\\\":false,\\\"is_enabled_translation\\\":false,\\\"max_num_tran_langs_allow_selecting\\\":0,\\\"allowed_speech_langs\\\":[],\\\"allowed_speech_users\\\":[],\\\"allowed_trans_langs\\\":[]},\\\"end_to_end_encryption_features\\\":{\\\"is_enabled\\\":false}},\\\"default_lock_settings\\\":{\\\"lock_screen_sharing\\\":true,\\\"lock_whiteboard\\\":true,\\\"lock_shared_notepad\\\":true},\\\"copyright_conf\\\":{\\\"display\\\":true,\\\"text\\\":\\\"Powered by <a href=\\\\\\\"https://www.plugnmeet.org\\\\\\\" target=\\\\\\\"_blank\\\\\\\">plugNmeet</a>\\\"},\\\"metadata_id\\\":\\\"3d5f55c2-e254-48f0-8642-d0fe38d0d850\\\"}\"", "duration": "2.278114ms", "status": "200"}
server-plugnmeet-api-1    | 08:21:53 | 200 |   93.911496ms | 172.18.0.1 | POST | /auth/room/create | -
server-livekit-1          | 2024-02-27T08:21:53.807Z    DEBUG   livekit.room    service/roommanager.go:778      updating room   {"room": "room01", "roomID": "RM_9okKDA5MadUb"}
server-plugnmeet-api-1    | 08:21:53 | 204 |      63.409µs | 172.18.0.1 | OPTIONS | /auth/room/getJoinToken | -
server-livekit-1          | 2024-02-27T08:21:53.808Z    DEBUG   livekit.psrpc.Room.UpdateRoomMetadata   psrpc/logging.go:69     client response {"topic": ["room01"], "request": {"room": "room01", "metadata": "{\"room_title\":\"Demo room\",\"welcome_message\":\"Welcome to plugNmeet!<br /> To share microphone click mic icon from bottom left side.\",\"is_recording\":false,\"is_active_rtmp\":false,\"parent_room_id\":\"\",\"is_breakout_room\":false,\"started_at\":\"1709022113\",\"room_features\":{\"allow_webcams\":true,\"mute_on_start\":false,\"allow_screen_share\":true,\"allow_rtmp\":true,\"allow_view_other_webcams\":true,\"allow_view_other_users_list\":true,\"admin_only_webcams\":false,\"allow_polls\":true,\"room_duration\":\"0\",\"enable_analytics\":true,\"allow_virtual_bg\":true,\"allow_raise_hand\":true,\"recording_features\":{\"is_allow\":true,\"is_allow_cloud\":true,\"enable_auto_cloud_recording\":false,\"is_allow_local\":true,\"only_record_admin_webcams\":false},\"chat_features\":{\"allow_chat\":true,\"allow_file_upload\":true,\"allowed_file_types\":[\"jpg\",\"png\",\"zip\"],\"max_file_size\":\"10\"},\"shared_note_pad_features\":{\"allowed_shared_note_pad\":true,\"is_active\":false,\"visible\":false,\"node_id\":\"\",\"host\":\"\",\"note_pad_id\":\"\",\"read_only_pad_id\":\"\"},\"whiteboard_features\":{\"allowed_whiteboard\":true,\"visible\":false,\"whiteboard_file_id\":\"\",\"file_name\":\"\",\"file_path\":\"\",\"total_pages\":0},\"external_media_player_features\":{\"allowed_external_media_player\":true,\"is_active\":false},\"waiting_room_features\":{\"is_active\":true,\"waiting_room_msg\":\"\"},\"breakout_room_features\":{\"is_allow\":true,\"is_active\":false,\"allowed_number_rooms\":6},\"display_external_link_features\":{\"is_allow\":true,\"is_active\":false},\"ingress_features\":{\"is_allow\":true,\"input_type\":\"RTMP_INPUT\",\"url\":\"\",\"stream_key\":\"\"},\"speech_to_text_translation_features\":{\"is_allow\":false,\"is_allow_translation\":true,\"is_enabled\":false,\"is_enabled_translation\":false,\"max_num_tran_langs_allow_selecting\":0,\"allowed_speech_langs\":[],\"allowed_speech_users\":[],\"allowed_trans_langs\":[]},\"end_to_end_encryption_features\":{\"is_enabled\":false}},\"default_lock_settings\":{\"lock_screen_sharing\":true,\"lock_whiteboard\":true,\"lock_shared_notepad\":true},\"copyright_conf\":{\"display\":true,\"text\":\"Powered by <a href=\\\"https://www.plugnmeet.org\\\" target=\\\"_blank\\\">plugNmeet</a>\"},\"metadata_id\":\"75ed455d-1419-49fd-bad2-ad32efa7e9b9\"}"}, "response": {"sid": "RM_9okKDA5MadUb", "name": "room01", "emptyTimeout": 7200, "creationTime": 1709022113, "turnPassword": "5EBYzDUwqeb8EHZoZCJ8HoDQIniK4exwiWDzn4AnPvk", "enabledCodecs": [{"mime": "audio/opus"}, {"mime": "audio/red"}, {"mime": "video/vp8"}, {"mime": "video/vp9"}, {"mime": "video/av1"}, {"mime": "video/h264"}], "metadata": "{\"room_title\":\"Demo room\",\"welcome_message\":\"Welcome to plugNmeet!<br /> To share microphone click mic icon from bottom left side.\",\"is_recording\":false,\"is_active_rtmp\":false,\"parent_room_id\":\"\",\"is_breakout_room\":false,\"started_at\":\"1709022113\",\"room_features\":{\"allow_webcams\":true,\"mute_on_start\":false,\"allow_screen_share\":true,\"allow_rtmp\":true,\"allow_view_other_webcams\":true,\"allow_view_other_users_list\":true,\"admin_only_webcams\":false,\"allow_polls\":true,\"room_duration\":\"0\",\"enable_analytics\":true,\"allow_virtual_bg\":true,\"allow_raise_hand\":true,\"recording_features\":{\"is_allow\":true,\"is_allow_cloud\":true,\"enable_auto_cloud_recording\":false,\"is_allow_local\":true,\"only_record_admin_webcams\":false},\"chat_features\":{\"allow_chat\":true,\"allow_file_upload\":true,\"allowed_file_types\":[\"jpg\",\"png\",\"zip\"],\"max_file_size\":\"10\"},\"shared_note_pad_features\":{\"allowed_shared_note_pad\":true,\"is_active\":false,\"visible\":false,\"node_id\":\"\",\"host\":\"\",\"note_pad_id\":\"\",\"read_only_pad_id\":\"\"},\"whiteboard_features\":{\"allowed_whiteboard\":true,\"visible\":false,\"whiteboard_file_id\":\"\",\"file_name\":\"\",\"file_path\":\"\",\"total_pages\":0},\"external_media_player_features\":{\"allowed_external_media_player\":true,\"is_active\":false},\"waiting_room_features\":{\"is_active\":true,\"waiting_room_msg\":\"\"},\"breakout_room_features\":{\"is_allow\":true,\"is_active\":false,\"allowed_number_rooms\":6},\"display_external_link_features\":{\"is_allow\":true,\"is_active\":false},\"ingress_features\":{\"is_allow\":true,\"input_type\":\"RTMP_INPUT\",\"url\":\"\",\"stream_key\":\"\"},\"speech_to_text_translation_features\":{\"is_allow\":false,\"is_allow_translation\":true,\"is_enabled\":false,\"is_enabled_translation\":false,\"max_num_tran_langs_allow_selecting\":0,\"allowed_speech_langs\":[],\"allowed_speech_users\":[],\"allowed_trans_langs\":[]},\"end_to_end_encryption_features\":{\"is_enabled\":false}},\"default_lock_settings\":{\"lock_screen_sharing\":true,\"lock_whiteboard\":true,\"lock_shared_notepad\":true},\"copyright_conf\":{\"display\":true,\"text\":\"Powered by <a href=\\\"https://www.plugnmeet.org\\\" target=\\\"_blank\\\">plugNmeet</a>\"},\"metadata_id\":\"3d5f55c2-e254-48f0-8642-d0fe38d0d850\"}"}, "duration": "1.894251ms"}
server-livekit-1          | 2024-02-27T08:21:53.808Z    INFO    livekit.api     service/twirp.go:124    API RoomService.UpdateRoomMetadata      {"service": "RoomService", "method": "UpdateRoomMetadata", "room": "room01", "size": 2108, "duration": "45.226317ms", "status": "200"}
server-livekit-1          | 2024-02-27T08:21:53.856Z    DEBUG   livekit.room    service/roommanager.go:761      api send data   {"room": "room01", "roomID": "RM_9okKDA5MadUb", "size": 2199}
server-livekit-1          | 2024-02-27T08:21:53.857Z    DEBUG   livekit.psrpc.Room.SendData     psrpc/logging.go:69     client response {"topic": ["room01"], "request": {"room": "room01", "data": "CAESJDRlOGZmZTQ4LWYwNzctNGVlYS05NmMwLWRhZWE4YmE3OWQxNzLsEAgZGh9UdWUsIDI3IEZlYiAyMDI0IA... (2.15kB)"}, "response": {}, "duration": "956.596µs"}
server-livekit-1          | 2024-02-27T08:21:53.857Z    INFO    livekit.api     service/twirp.go:124    API RoomService.SendData        {"service": "RoomService", "method": "SendData", "room": "room01", "size": 2199, "duration": "1.096792ms", "status": "200"}
server-livekit-1          | 2024-02-27T08:21:53.879Z    INFO    livekit.webhook webhook/url_notifier.go:88      sent webhook    {"url": "http://plugnmeet-api:8080/webhook", "event": "room_started", "eventDetails": {"event": "room_started", "room": {"sid": "RM_9okKDA5MadUb", "name": "room01", "emptyTimeout": 7200, "creationTime": 1709022113, "turnPassword": "5EBYzDUwqeb8EHZoZCJ8HoDQIniK4exwiWDzn4AnPvk", "enabledCodecs": [{"mime": "audio/opus"}, {"mime": "audio/red"}, {"mime": "video/vp8"}, {"mime": "video/vp9"}, {"mime": "video/av1"}, {"mime": "video/h264"}], "metadata": "{\"room_title\":\"Demo room\",\"welcome_message\":\"Welcome to plugNmeet!<br /> To share microphone click mic icon from bottom left side.\",\"is_recording\":false,\"is_active_rtmp\":false,\"parent_room_id\":\"\",\"is_breakout_room\":false,\"started_at\":\"1709022113\",\"room_features\":{\"allow_webcams\":true,\"mute_on_start\":false,\"allow_screen_share\":true,\"allow_rtmp\":true,\"allow_view_other_webcams\":true,\"allow_view_other_users_list\":true,\"admin_only_webcams\":false,\"allow_polls\":true,\"room_duration\":\"0\",\"enable_analytics\":true,\"allow_virtual_bg\":true,\"allow_raise_hand\":true,\"recording_features\":{\"is_allow\":true,\"is_allow_cloud\":true,\"enable_auto_cloud_recording\":false,\"is_allow_local\":true,\"only_record_admin_webcams\":false},\"chat_features\":{\"allow_chat\":true,\"allow_file_upload\":true,\"allowed_file_types\":[\"jpg\",\"png\",\"zip\"],\"max_file_size\":\"10\"},\"shared_note_pad_features\":{\"allowed_shared_note_pad\":true,\"is_active\":false,\"visible\":false,\"node_id\":\"\",\"host\":\"\",\"note_pad_id\":\"\",\"read_only_pad_id\":\"\"},\"whiteboard_features\":{\"allowed_whiteboard\":true,\"visible\":false,\"whiteboard_file_id\":\"\",\"file_name\":\"\",\"file_path\":\"\",\"total_pages\":0},\"external_media_player_features\":{\"allowed_external_media_player\":true,\"is_active\":false},\"waiting_room_features\":{\"is_active\":true,\"waiting_room_msg\":\"\"},\"breakout_room_features\":{\"is_allow\":true,\"is_active\":false,\"allowed_number_rooms\":6},\"display_external_link_features\":{\"is_allow\":true,\"is_active\":false},\"ingress_features\":{\"is_allow\":true,\"input_type\":\"RTMP_INPUT\",\"url\":\"\",\"stream_key\":\"\"},\"speech_to_text_translation_features\":{\"is_allow\":false,\"is_allow_translation\":true,\"is_enabled\":false,\"is_enabled_translation\":false,\"max_num_tran_langs_allow_selecting\":0,\"allowed_speech_langs\":[],\"allowed_speech_users\":[],\"allowed_trans_langs\":[]},\"end_to_end_encryption_features\":{\"is_enabled\":false}},\"default_lock_settings\":{\"lock_screen_sharing\":true,\"lock_whiteboard\":true,\"lock_shared_notepad\":true},\"copyright_conf\":{\"display\":true,\"text\":\"Powered by <a href=\\\"https://www.plugnmeet.org\\\" target=\\\"_blank\\\">plugNmeet</a>\"},\"metadata_id\":\"3d5f55c2-e254-48f0-8642-d0fe38d0d850\"}"}, "id": "EV_eusjQzs3tX39", "createdAt": 1709022113}}
server-plugnmeet-api-1    | 08:21:53 | 200 |    123.4171ms | 172.18.0.4 | POST | /webhook | -
server-plugnmeet-api-1    | 08:21:53 | 200 |   67.866868ms | 172.18.0.1 | POST | /auth/room/getJoinToken | -
server-livekit-1          | 2024-02-27T08:21:53.920Z    INFO    livekit.api     service/twirp.go:124    API RoomService.ListParticipants        {"service": "RoomService", "method": "ListParticipants", "room": "room01", "duration": "230.508µs", "status": "200"}
server-plugnmeet-api-1    | 08:21:56 | 204 |      35.246µs | 172.18.0.1 | OPTIONS | /api/verifyToken | -
server-livekit-1          | 2024-02-27T08:21:56.291Z    INFO    livekit.api     service/twirp.go:124    API RoomService.ListRooms       {"service": "RoomService", "method": "ListRooms", "room": ["room01"], "duration": "775.473µs", "status": "200"}
server-plugnmeet-api-1    | 08:21:56 | 200 |   55.778701ms | 172.18.0.1 | POST | /api/verifyToken | -
server-livekit-1          | 2024-02-27T08:21:58.868Z    DEBUG   livekit routing/signal.go:100   starting signal connection      {"room": "room01", "reqNodeID": "ND_LarFVep4LH9V", "participant": "1709020854343", "connID": "CO_JC4VAUB96W9U"}
server-livekit-1          | 2024-02-27T08:21:58.869Z    DEBUG   livekit service/roommanager.go:338      starting RTC session    {"room": "room01", "nodeID": "ND_LarFVep4LH9V", "participant": "1709020854343", "clientInfo": {"sdk": 1, "version": "1.15.10", "protocol": 11, "os": "Linux", "deviceModel": "Other", "browser": "Chrome", "browserVersion": "122.0.0", "address": "172.18.0.1"}, "reconnect": false, "reconnectReason": "RR_UNKNOWN", "adaptiveStream": true, "numParticipants": 0}
server-livekit-1          | 2024-02-27T08:21:58.870Z    DEBUG   livekit.room    rtc/room.go:405 new participant joined  {"room": "room01", "roomID": "RM_9okKDA5MadUb", "pID": "PA_oHiJAnGbix26", "participant": "1709020854343", "clientInfo": {"sdk": 1, "version": "1.15.10", "protocol": 11, "os": "Linux", "deviceModel": "Other", "browser": "Chrome", "browserVersion": "122.0.0", "address": "172.18.0.1"}, "options": {"AutoSubscribe":true}, "numParticipants": 0}
server-livekit-1          | 2024-02-27T08:21:58.870Z    DEBUG   livekit rtc/participant.go:1247 updating participant state      {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "state": "JOINED"}
server-livekit-1          | 2024-02-27T08:21:58.870Z    DEBUG   livekit rtc/participant.go:879  SetMigrateState {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "state": "MIGRATE_STATE_COMPLETE"}
server-livekit-1          | 2024-02-27T08:21:58.871Z    DEBUG   livekit.transport       rtc/transport.go:579    ice gathering state change      {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "transport": "SUBSCRIBER", "state": "gathering"}
server-livekit-1          | 2024-02-27T08:21:58.871Z    DEBUG   livekit.sub     rtc/participant.go:1291 sending offer   {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "transport": "SUBSCRIBER"}
server-livekit-1          | 2024-02-27T08:21:58.871Z    INFO    livekit service/rtcservice.go:309       new client WS connected {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "connID": "CO_JC4VAUB96W9U", "reconnect": false, "reconnectReason": "RR_UNKNOWN", "adaptiveStream": true, "selectedNodeID": "ND_LarFVep4LH9V"}
server-livekit-1          | 2024-02-27T08:21:58.871Z    DEBUG   livekit service/rtcservice.go:348       sending offer   {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "offer": {"Offer":{"type":"offer","sdp":"v=0\r\no=- 8221554341007088357 1709022118 IN IP4 0.0.0.0\r\ns=-\r\nt=0 0\r\na=fingerprint:sha-256 E9:01:43:D1:4C:34:AD:30:58:34:56:51:98:4A:7B:14:B8:35:FE:EF:69:BA:DC:B1:09:37:EF:91:C8:03:AB:8B\r\na=extmap-allow-mixed\r\na=group:BUNDLE 0\r\nm=application 9 UDP/DTLS/SCTP webrtc-datachannel\r\nc=IN IP4 0.0.0.0\r\na=setup:actpass\r\na=mid:0\r\na=sendrecv\r\na=sctp-port:5000\r\na=ice-ufrag:EqqaCIMEYljCrShO\r\na=ice-pwd:oZeypdjWhnOJfcnWfeQjYUGXneTEiANe\r\n"}}}
server-livekit-1          | 2024-02-27T08:21:58.941Z    DEBUG   livekit service/rtcservice.go:428       received answer {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "answer": {"Answer":{"type":"answer","sdp":"v=0\r\no=- 348764461506487883 2 IN IP4 127.0.0.1\r\ns=-\r\nt=0 0\r\na=extmap-allow-mixed\r\na=group:BUNDLE 0\r\nm=application 9 UDP/DTLS/SCTP webrtc-datachannel\r\nc=IN IP4 0.0.0.0\r\na=setup:active\r\na=mid:0\r\na=ice-ufrag:rVZ8\r\na=ice-pwd:ZBm3FUUtUH4RqtWSSY759Je7\r\na=fingerprint:sha-256 55:35:F9:66:1C:87:06:A5:CA:20:C2:E4:E7:26:04:21:B3:33:08:CE:67:CB:C6:A0:2A:4D:91:0D:29:42:19:74\r\na=ice-options:trickle\r\na=sctp-port:5000\r\n"}}}
server-livekit-1          | 2024-02-27T08:21:58.942Z    DEBUG   livekit.sub     rtc/participant.go:604  received answer {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "transport": "SUBSCRIBER"}
server-livekit-1          | 2024-02-27T08:21:58.942Z    DEBUG   livekit.transport       rtc/transport.go:619    ice connection state change     {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "transport": "SUBSCRIBER", "state": "checking"}
server-livekit-1          | 2024-02-27T08:21:58.943Z    DEBUG   livekit.transport       rtc/transport.go:643    peer connection state change    {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "transport": "SUBSCRIBER", "state": "connecting"}
server-livekit-1          | 2024-02-27T08:21:58.945Z    DEBUG   livekit.transport       rtc/transportmanager.go:450     ignoring mDNS candidate {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "candidate": "2021237206 1 udp 2113937151 c3903939-9630-4532-882b-37b291eef69f.local 55173 typ host generation 0 ufrag rVZ8 network-cost 999", "target": "SUBSCRIBER"}
server-livekit-1          | 2024-02-27T08:21:58.970Z    DEBUG   livekit.transport       rtc/transport.go:579    ice gathering state change      {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "transport": "SUBSCRIBER", "state": "complete"}
server-livekit-1          | 2024-02-27T08:22:13.934Z    INFO    livekit service/rtcservice.go:269       finishing WS connection {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "connID": "CO_JC4VAUB96W9U", "closedByClient": true}
server-livekit-1          | 2024-02-27T08:22:13.934Z    DEBUG   livekit service/rtcservice.go:335       nothing to read from response source    {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "connID": "CO_JC4VAUB96W9U"}
server-livekit-1          | 2024-02-27T08:22:13.935Z    DEBUG   livekit service/signal.go:168   signal stream closed    {"room": "room01", "participant": "1709020854343", "connID": "CO_JC4VAUB96W9U", "error": null}
server-livekit-1          | 2024-02-27T08:22:13.935Z    DEBUG   livekit.transport       rtc/transportmanager.go:801     signal source valid     {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "valid": false}
server-livekit-1          | 2024-02-27T08:22:13.935Z    INFO    livekit rtc/participant.go:741  participant closing     {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "sendLeave": false, "reason": "JOIN_FAILED", "isExpectedToResume": false}
server-livekit-1          | 2024-02-27T08:22:13.935Z    DEBUG   livekit rtc/participant.go:1247 updating participant state      {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "state": "DISCONNECTED"}
server-livekit-1          | 2024-02-27T08:22:13.935Z    INFO    livekit rtc/room.go:545 removing participant without connection {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "subscriberCandidates": ["[local] udp4 host 172.18.0.4:7882", "[local] tcp4 host 172.18.0.4:7881", "[local] udp4 srflx 2.42.32.137:52657 related 0.0.0.0:52087", "[remote] udp4 srflx 2.42.32.137:55173 related 0.0.0.0:0"], "connectionType": "unknown", "reason": "STATE_DISCONNECTED"}
server-livekit-1          | 2024-02-27T08:22:13.936Z    DEBUG   livekit service/roommanager.go:590      RTC session finishing   {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "connID": "CO_JC4VAUB96W9U"}
server-livekit-1          | 2024-02-27T08:22:13.936Z    DEBUG   livekit routing/signal.go:136   signal stream closed    {"room": "room01", "reqNodeID": "ND_LarFVep4LH9V", "participant": "1709020854343", "connID": "CO_JC4VAUB96W9U", "error": "stream closed"}
server-livekit-1          | 2024-02-27T08:22:13.936Z    DEBUG   livekit.transport       rtc/transport.go:643    peer connection state change    {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "transport": "PUBLISHER", "state": "closed"}
server-livekit-1          | 2024-02-27T08:22:13.936Z    DEBUG   livekit.transport       rtc/transport.go:1420   leaving events processor        {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "transport": "SUBSCRIBER"}
server-livekit-1          | 2024-02-27T08:22:13.936Z    DEBUG   livekit.transport       rtc/transport.go:1420   leaving events processor        {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "transport": "PUBLISHER"}
server-livekit-1          | 2024-02-27T08:22:13.936Z    DEBUG   livekit.transport       rtc/transport.go:579    ice gathering state change      {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "transport": "SUBSCRIBER", "state": "closed"}
server-livekit-1          | 2024-02-27T08:22:13.936Z    DEBUG   livekit.transport       rtc/transport.go:619    ice connection state change     {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "transport": "SUBSCRIBER", "state": "closed"}
server-livekit-1          | 2024-02-27T08:22:13.936Z    DEBUG   livekit.transport       rtc/transport.go:643    peer connection state change    {"room": "room01", "roomID": "RM_9okKDA5MadUb", "participant": "1709020854343", "pID": "PA_oHiJAnGbix26", "remote": false, "transport": "SUBSCRIBER", "state": "closed"}`

**Reply by ravalosv** [reply]
On Windows I solved it using 127.0.0.1 (for a local installation)

**Comment by deenodj** [comment]
I thing i have teh same problem, where did you edit the good url or ipadres, its file?
I have running plugnmeet localy in a docker, there is no firewall (turned off), when i acces the login, i acces my microphone settings, than he trying to connect and the printscreen comes up.
Seems that he can't connect.... 

![err](https://github.com/user-attachments/assets/509374d8-6369-4067-9ba5-a541c702c5be)
